### ISSUE TITLE
LAION-400M index

### ISSUE THREAD

**Issue by xbldi** [rom1504/clip-retrieval#318]
Hi, thanks for this great repo!

I'd like to ask if you could share the LAION-400M index used for the demo website https://rom1504.github.io/clip-retrieval

So far the only public 400M index I can find is from here https://the-eye.eu/public/AI/cah/laion400m-indexes/. However, they don't have the metadata required to run clip-retrieval backend.

If you can't share the index, could you share the configs used to build this index with auto-faiss?

Thanks in advance!

**Comment by piotr-teterwak** [comment]
Agreed, thank you! And also the h-params of how it was built would be very useful.

**Comment by jvpassarelli** [comment]
Working on laion search website [at this url](https://d1o3cl9ccvfpb2.cloudfront.net/) - hope it is useful - [documenting effort here](https://github.com/jvpassarelli/laion-image-search)
